types/braintree-web/modules/apple-pay.d.ts

```diff
@@ -72,6 +73,12 @@ export interface ApplePayDetails {
     dpanLastTwo: string;
 }
 
+export interface ApplePayLineItem {
+    type: string;
+    label: string;
+    amount: number;
```

Comment:
Looking at the [`ApplePayLineItem` definition](https://developer.apple.com/documentation/apple_pay_on_the_web/applepaylineitem), it looks like [`type`](https://developer.apple.com/documentation/apple_pay_on_the_web/applepaylineitemtype) should be enum or type alias with predefined options. 

Also, could you populate the remaining fields in this structure? There are several missing.